README.md

```diff
@@ -6,6 +6,10 @@ In this project, I will build a Single Page App (SPA), I will create an MVP vers
 - Add a book.
 - Remove a selected book.
 
+## Live demo link
+
+- Netlify live demo [link](https://reactappbookstore.netlify.app)
+
 ## Getting Started
```

Comment:
- [ ] Your app looks great and I think you need to be proud of it 💯. Let's add a preview or screenshot of your app to your README so that users get to see your awesome design. What do you think?

Reply:
Yes, sure because the live demo link displays the previous merged branch 😉